js_modules/dagit/src/index.tsx

```diff
@@ -33,9 +35,11 @@ if (process.env.REACT_APP_RENDER_API_RESULTS) {
   );
 } else {
   ReactDOM.render(
-    <ApolloProvider client={client}>
-      <App />
-    </ApolloProvider>,
+    <WebsocketContext.Provider value={websocketClient}>
+      <ApolloProvider client={client}>
```

Comment:
I spent a little while trying to find a way to pull the websocket out of the Apollo client but it seems to wrap / decorate the `link` you provide and hides the original. I figured exposing the websocket client itself to the app was the easiest way - honestly we could probably just set `window.websocket` too...